shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-api/src/main/java/org/apache/shardingsphere/data/pipeline/api/check/consistency/yaml/YamlDataConsistencyCheckResult.java

```diff
@@ -33,9 +33,9 @@
 @Getter
 @Setter
 @ToString
-public final class YamlDataConsistencyCheckResultConfiguration {
+public final class YamlDataConsistencyCheckResult {
```

Comment:
`YamlDataConsistencyCheckResult` has different hierarchy with `DataConsistencyCheckResult`, it's better to improve it